pkg/catalog/catalog.go

```diff
@@ -87,13 +90,40 @@ func VersionExists(client *occlient.Client, componentType string, componentVersi
 }
 
 // getDefaultBuilderImages returns the default builder images available in the
-// openshift namespace
+// openshift and the current namespaces
 func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
-	imageStreams, err := client.GetImageStreams(occlient.OpenShiftNameSpace)
-	if err != nil {
-		return nil, errors.Wrap(err, "unable to get Image Streams")
+	var imageStreams []imagev1.ImageStream
+	currentNamespace := client.GetCurrentProjectName()
+	var err error
+
+	// Fetch imagestreams from default openshift namespace
+	openshiftNSImageStreams, possibleError := client.GetImageStreams(occlient.OpenShiftNameSpace)
+	if possibleError != nil {
+		// Tolerate the error as it might only be a partial failure
+		// We may get the imagestreams from other Namespaces
+		err = errors.Wrapf(possibleError, "unable to get Image Streams from namespace %s", occlient.OpenShiftNameSpace)
+		// log it for debugging purposes
+		glog.V(4).Infof("Unable to get Image Streams from namespace %s. Error %s", occlient.OpenShiftNameSpace, possibleError.Error())
 	}
 
+	// Fetch imagestreams from current namespace
+	currentNSImageStreams, possibleError := client.GetImageStreams(currentNamespace)
```

Comment:
again, `possibleError` is unconventional, use `err`

Reply:
Ok I tried to simplify it this time..
Hope that looks ok